### PR TITLE
Update changelog

### LINKED ISSUE #595: [Docs] : Update CHANGELOG.md
**Which document needs to be updated?**
We have to update [changelog](https://github.com/OT-CONTAINER-KIT/redis-operator/blob/master/CHANGELOG.md) 

**Expected changes**
Since we are on v0.15.0 and the log only covers till the v0.13.0. We need to update the change log from the [release page](https://github.com/OT-CONTAINER-KIT/redis-operator/releases). 

**What version of redis-operator are you using?**

redis-operator version: v0.15.0

### PR DESCRIPTION
**Pull request by Akshaykumar05**
**Description**

Fixes #595

**Type of change**

* Update the changelog

### CODE REVIEW

File: .github/workflows/operator-ci.yaml
Comment (shubham-cmyk) [inline .github/workflows/operator-ci.yaml:1]: Discard this file

**Review by shubham-cmyk — COMMENTED** [review]
Name of PR is wrong

**Comment by shubham-cmyk** [comment]
@Akshaykumar05 Any update

**Comment by Akshaykumar05** [comment]
> @Akshaykumar05 Any update
@shubham-cmyk 
After giving signed-off and pushing command, I am getting this issue of conflict merge and it is asking to solve merge conflict manually and mark them resolved.
dco issue

**Comment by sadath-12** [comment]
@Akshaykumar05 revert back to where you made this pr and then use command ```git commit -s ```

**Review by shubham-cmyk — APPROVED** [review]
LGTM

**Pull request merged**